reference-implementation/lib/writable-stream.js

```diff
@@ -130,27 +136,17 @@ function WritableStreamAbort(stream, reason) {
   if (state === 'errored') {
     return Promise.reject(stream._storedError);
   }
+  if (stream._pendingAbortRequest !== undefined) {
+    return Promise.reject(new TypeError('Aborting in progress'));
+  }
 
   assert(state === 'writable', 'state must be writable');
 
   const error = new TypeError('Requested to abort');
-  if (stream._pendingAbortRequest !== undefined) {
+  if (stream._pendingAbortRequest === true) {
```

Comment:
It's a promise. Well, actually a record containing a promise and a reason. Great, now I need to work out whether this is a test gap or a no-op.